### PR TITLE
ICU-21249 integrate CLDR release-38-beta2 to ICU trunk

### PR DESCRIPTION
**Pull request by pedberg-icu**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21249
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [x] Tests included
- [ ] Documentation is changed or added

Integrated CLDR release-38-beta2 to ICU trunk. The only ICU data update is a chnage to the samples for French plural rules. However, this causes a test failure in PluralRulesTest.java, which I have skipped with a logKnownIssue on a new ticket for this: https://unicode-org.atlassian.net/browse/ICU-21328

### CODE REVIEW

File: icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java [hunk 1]
```diff
@@ -399,7 +399,10 @@ public void testUniqueRules() {
                             }
                             String old = collisionTest.get(item);
                             if (old != null) {
-                                errln(locale + "\tNon-unique rules: " + item + " => " + old + " & " + foundKeyword);
+                                if (!locale.getLanguage().equals("fr") ||
+                                        !logKnownIssue("21328", "fr Non-unique rules: 1e6 => one & many0")) {
```
Comment (markusicu) [inline icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java:403]: Is that a typo? many0 --> many?
(Could use `git commit --amend` to fix it and keep it to a single commit.)

**Review by markusicu — DISMISSED** [review]
rslgtm but fixing the typo would be nice

**Review by jefgen — APPROVED** [review]
rslgtm

File: icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java (hunk 1 above)
Comment (echeran) [inline icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java:403]: This looks fine to me.  I marked the new ticket ICU-21328 and the ticket ICU-21322 (replace FixedDecimal with DecimalQuantity for plural rule samples) as related.

**Review by echeran — APPROVED** [review]
LGTM.

**Review by markusicu — APPROVED** [review]
lgtm tnx

**Comment by pedberg-icu** [comment]
> rslgtm but fixing the typo would be nice

Thanks, I fixed the typo, no other changes

**Pull request merged**